.github/workflows/ci.yaml

```diff
@@ -8,7 +8,7 @@ on:
     types: [published]
   workflow_dispatch:
   repository_dispatch:
-    types: [start-ci]
+    types: [start-ci, demo-deploy]
```

Comment:
ocd: as the variable is named `DEPLOY_DEMO` we could call this `deploy-demo` instead of `demo-deploy`